calculator.py

```diff
@@ -0,0 +1,54 @@
+import data_structures
+
+
+class Calculator:
+
+    def __init__(self,weather_data):
+        self.weather_data: list(data_structures.WeatherData)=weather_data
+
+    @staticmethod
+    def _year_and_data_validation(data_to_be_compared, data_year, year_to_be_compared):
+        return True if data_year == year_to_be_compared and data_to_be_compared else False
+
+    @staticmethod
+    def _comparison_operation(operation_description, data_to_be_compared, value):
```

Comment:
there should be only one return statement in a fuction. refactor this where needed